lh-plugin-demo.cabal

```diff
@@ -20,10 +20,17 @@ library
   hs-source-dirs:
       src
   build-depends:
-      liquid-ghc-prim,
-      liquid-base,
       liquid-prelude,
-      liquid-containers,
+      liquid-vector,
       liquidhaskell
+  if impl(ghc >= 9.2)
+    build-depends:
+      base,
+      containers,
+      vector
+  else
+    build-depends:
+      liquid-base,
+      liquid-containers
```

Comment:
This bit becomes a bit less trivial now that I'm trying to support building with two different GHCs.